2.4/alpine/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM alpine:3.4
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN addgroup -g 82 -S www-data \
+	&& adduser -u 82 -D -S -G www-data www-data
+
+ENV HTTPD_PREFIX /usr/local/apache2
+ENV PATH $PATH:$HTTPD_PREFIX/bin
```

Comment:
If we move `$HTTPD_PREFIX/bin` to be pre-pended, won't that solve the "wrong httpd" problem? (and allow us to use `exec httpd ...` in `httpd-foreground`)

```dockerfile
ENV HTTPD_PREFIX /usr/local/apache2
ENV PATH $HTTPD_PREFIX/bin:$PATH
```